public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/SimplifiedRouting.tsx

```diff
@@ -0,0 +1,258 @@
+import { css } from '@emotion/css';
+import { AnyAction, createAction, createReducer } from '@reduxjs/toolkit';
+import React, { useEffect, useMemo, useReducer } from 'react';
+import { useFormContext } from 'react-hook-form';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { SelectableValue } from '@grafana/data/src/types';
+import { Field, Icon, Select, Stack, Text, useStyles2 } from '@grafana/ui';
+import { onCallApi } from 'app/features/alerting/unified/api/onCallApi';
+import { useAlertmanagerConfig } from 'app/features/alerting/unified/hooks/useAlertmanagerConfig';
+import { usePluginBridge } from 'app/features/alerting/unified/hooks/usePluginBridge';
+import { INTEGRATION_ICONS } from 'app/features/alerting/unified/types/contact-points';
+import { SupportedPlugin } from 'app/features/alerting/unified/types/pluginBridges';
+import { RuleFormValues } from 'app/features/alerting/unified/types/rule-form';
+import { extractReceivers } from 'app/features/alerting/unified/utils/receivers';
+import { createUrl } from 'app/features/alerting/unified/utils/url';
+import { Receiver } from 'app/plugins/datasource/alertmanager/types';
+
+import { ReceiverMetadataBadge } from '../../../receivers/grafanaAppReceivers/ReceiverMetadataBadge';
+import { ReceiverTypes } from '../../../receivers/grafanaAppReceivers/onCall/onCall';
+import { ReceiverPluginMetadata, getOnCallMetadata } from '../../../receivers/grafanaAppReceivers/useReceiversMetadata';
+import {
+  AlertManagerMetaData,
+  useGetAlertManagersMetadata,
+} from '../../notificaton-preview/useGetAlertManagersSourceNamesAndImage';
+
+export interface AMContactPoint {
+  alertManager: AlertManagerMetaData;
+  selectedContactPoint?: string;
+}
+
+export const selectContactPoint = createAction<{ receiver: string | undefined; alertManager: AlertManagerMetaData }>(
+  'simplifiedRouting/selectContactPoint'
+);
+
+export const receiversReducer = createReducer<AMContactPoint[]>([], (builder) => {
+  builder.addCase(selectContactPoint, (state, action) => {
+    const { receiver, alertManager } = action.payload;
+    const newContactPoint: AMContactPoint = { selectedContactPoint: receiver, alertManager };
+    const existingContactPoint = state.find((cp) => cp.alertManager.name === alertManager.name);
+
+    if (existingContactPoint) {
+      existingContactPoint.selectedContactPoint = receiver;
+    } else {
+      state.push(newContactPoint);
```

Comment:
I think this is not possible because we create the state from the list of possible alert managers...but just in case?